recipes/conversation_view/feed_pill/feed_item_pill.test.js

```diff
@@ -1,6 +1,6 @@
 import { mount } from '@vue/test-utils';
 import DtRecipeFeedItemPill from './feed_item_pill.vue';
-import { beforeEach } from 'vitest';
+import { beforeEach, describe } from 'vitest';
```

Comment:
Not sure if `describe` or `beforeEach` is actually needed, noticed that runs ok without it.

Reply:
They're not needed, please remove them.